www/strf03/bandCamp/band_registration.php

```diff
@@ -0,0 +1,204 @@
+                    <?php echo implode('<br>', array_values($errors)); ?>
+                </div>
+            <?php endif; ?>
+            <form method="POST" enctype="multipart/form-data">
+                <div class="mb-3">
+                    <label for="username">Jméno kapely</label>
+                    <div class="input-group">
+
+                        <input name="band_name" type="text" class="form-control" placeholder="Jméno kapely"
+                               value="<?php echo @$band_name; ?>">
+                    </div>
+                </div>
+
+                <div class="mb-3">
+                    <label for="email">Email</label>
+                    <input type="email" name="email" class="form-control" placeholder="Email"
+                           value="<?php echo @$email; ?>">
+                </div>
+                <div class="mb-3">
+                    <label for="music_genres">Hudební styly (podrž ctrl pro označení více)</label>
+                    <select name="music_genres[]" class="form-control" multiple="multiple">
+                        <?php foreach ($music_genres as $music_genre): ?>
+                            <option value="<?php echo $music_genre['music_genre_id']; ?>"><?php echo $music_genre['music_genre_name']; ?></option>
+                        <?php endforeach; ?>
+                    </select>
+
+                </div>
+                <div class="form-group"> <!-- Date input -->
+                    <label class="control-label" for="date">Datum vzniku kapely</label>
+                    <input class="form-control" name="date_started" placeholder="YYYY-MM-DD" type="text"
+                           value="<?php echo @$date_started; ?>"/>
+                </div>
+
+
+                <div class="mb-3">
+                    <label for="district">Kraj</label>
+                    <select name="district" class="custom-select d-block w-100">
+                        <option value="">Vyber kraj...</option>
+                        <?php foreach ($districts as $name): ?>
+                            <option value="<?php echo $name ?>"> <?php echo $name ?> </option>
+                        <?php endforeach; ?>
+                    </select>
+                </div>
+
+                <div class="mb-3">
+                    <label for="password">Heslo</label>
+                    <input name="password" type="password" class="form-control" placeholder="Heslo">
+                </div>
+                <div class="mb-3">
+                    <label for="password_again">Heslo znovu</label>
+                    <input name="password_again" type="password" class="form-control"
+                           placeholder="Heslo ještě jednou">
+                </div>
+
+
+                <div class="mb-3">
+                    <label for="image">Avatar</label>
+                    <input name="image" type="file" class="form-control-input">
```

Comment:
chybi mime type

Reply:
To znamená atribut accept? Např.
<input name="image" type="file" class="form-control-input" accept="image/x-png,image/gif,image/jpeg">

Reply:
jop